src/components/shared/input/Input.tsx

```diff
@@ -0,0 +1,14 @@
+import "./Input.scss";
+
+export interface InputProps {
+  labelTitle: string;
+}
+
+export const Input = ({ labelTitle }: InputProps) => {
+  return (
+    <label className="label" htmlFor={labelTitle}>
+      {labelTitle}
+      <input id={labelTitle} placeholder={labelTitle} />
```

Comment:
I think that placeholder should be a separate prop

Reply:
after discussion we agreed to leave it for now as it is.